sdk/core/Azure.Core/tests/RequestContentHelperTests.cs

```diff
@@ -27,6 +27,11 @@ public static IEnumerable<TestCaseData> GetOneDateTimeData()
             yield return new TestCaseData(DateTimeOffset.Parse("2022-08-26T18:38:00Z"));
         }
 
+        public static IEnumerable<TestCaseData> GetBinaryDataData()
+        {
+            yield return new TestCaseData(BinaryData.FromString("\"test\""));
```

Comment:
Could we have more test cases to include other cases in binary data, like other primitive types, array and json object?

Reply:
Thanks @ArcturusZhang I added.